README.md

```diff
@@ -365,8 +367,9 @@ Services to securely store your Docker images.
 * [proxy](https://github.com/factorish/proxy) - lightweight nginx based load balancer self using service discovery provided by registrator. by [@factorish](https://github.com/factorish)
 * [registrator](https://github.com/progrium/registrator) - Service registry bridge for Docker by [@progrium][progrium]
 
-## Services for running containers
+## CaaS (Containers as a Service) - Services for running/hosting containers
```

Comment:
If you change the name of the link, you have to change it also in the menu....
I'd recommend to see the result in github and test the link. 
Ping here if you need help 😄